eip-0024.md

```diff
@@ -7,24 +7,50 @@
 * Forking: not needed 
 
 ## Motivation 
-With the discovery that we can easily use spent boxes in contracts ([see here](https://www.ergoforum.org/t/ergoscript-design-patterns/222/23?u=anon_real)), some new features are introduced for artworks and can be extended furthur in the future. This EIP is an extension of [Asset standard](eip-0004.md) and tents to standardize artwrok issuance in particular.
+With the discovery that we can easily use spent boxes in contracts ([see here](https://www.ergoforum.org/t/ergoscript-design-patterns/222/23?u=anon_real)), some new features are introduced for artworks and can be extended further in the future. This EIP is an extension of [Asset standard](eip-0004.md) and tents to standardize artwork issuance in particular.
 
 
-## Design
+## Design V1
 There are two important boxes in the issuance process of an artwork.
 - The issuance box, which follows [Asset standard](eip-0004.md) depending on the type of artwork.
 - The issuer box, which is the first input of the transaction that is issuing the artwork. In particular, the box with the same ID as the artwork's token ID.
 
 Now we will discuss why the issuer box is important and define a standard for it.
 
-The issuer box is important because it has the same ID as the artwork's token ID and we can include it (although it is spent) in our transactions either in registers or as context data. This means that we can prove/verify certain features that an artwork may have using this box. The following is the current standard for this box - which may be extended in the future.
+The issuer box is important because it has the same ID as the artwork's token ID and we can include it (although it is spent) in our transactions either in registers or as context data. This means that we can prove/verify certain features that an artwork may have using this box. The following is the v1 standard for this box - which may be extended in the future (refer to Design V2 part).
+
+- R4 of this box is an Integer, showing 1000 * royalty percentage of the artwork. e.g, 20 for 2% royalty. If R4 of the issuer box of an artowrk is empty or non-positive, then royalty is considered to be 0%.
+- In the current design of the [Auction contranct](eip-0022.md), proposition bytes of this box is the contract that the royalty percentage will be sent to. This is a suboptimal design because of AOT costing mechanism which is supposed to be replaced with JIT costing with a soft-fork that will happen with v5.0 of [ergo node](https://github.com/ergoplatform/ergo).
+  
+   In the case of a simple proxy contract, this means that the artist will receive the royalty share. However, the proxy contract may enforce the royalty to go to any other complex contract - e.g., 20% to the artist, and 80% to some charity.
```

Comment:
Does this feel a little out of context? As in there's no context leading into this statement.

Reply:
Please let me know if it is better now.